homework-submission/class16-week1/src/components/Todo.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+class Todo extends React.Component {
+  render() {
+    const { description, deadline } = this.props;
+    return (
+      <li className="todo_item">
+        <h2>
+          Description{" "}
```

Comment:
What is the point from having `{" "}` ? adding a space? this should be accomplished by CSS. add `margin-right` to `.todo_desc`

Reply:
This has been added by default as a result of using Prettier plugin in VS.